p2p/simulations/network.go

```diff
@@ -388,7 +399,7 @@ func (self *Network) Stop(id discover.NodeID) error {
 	node.Up = false
 	log.Info(fmt.Sprintf("stop node %v: %v", id, node.Up))
 
-	self.events.Send(ControlEvent(node))
+	self.events.Send(NewEvent(node))
```

Comment:
ah! well, because I wasn't receiving any node down live events, only node down control events! Is this fix incorrectly addressing this? Remember we are ignoring control events in the frontend